core/src/test/java/com/sequenceiq/cloudbreak/service/template/ClusterTemplateViewServiceTest.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.cloudbreak.service.template;
+
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+
+import com.sequenceiq.cloudbreak.controller.exception.BadRequestException;
+import com.sequenceiq.cloudbreak.domain.stack.cluster.ClusterTemplateView;
+
+public class ClusterTemplateViewServiceTest {
+
+    @InjectMocks
+    private ClusterTemplateViewService underTest;
+
+    @Before
+    public void setup() {
+        initMocks(this);
+    }
+
+    @Test(expected = BadRequestException.class)
```

Comment:
Use ExpectedException class instead of this. You cannot validate the message of error.